projects/plugins/crm/includes/ZeroBSCRM.GeneralFuncs.php

```diff
@@ -1553,4 +1553,61 @@ function jpcrm_youtube_url_to_video_id( $video_url ) {
 
 /* ======================================================
    / YouTube Helpers
-   ====================================================== */
\ No newline at end of file
+====================================================== */
+
+// These constants were created to simplify the process when this page gets
+// moved to a React-based approach. It was a choice to not refactor any further
+// than this.
```

Comment:
This looks like a multi-line comment 😄 

> ```
> /*
>  * This is a comment that is long enough to warrant being stretched over
>  * the span of multiple lines. You'll notice this follows basically
>  * the same format as the PHPDoc wrapping and comment block style.
>  */
> ```
> **Important note**: Multi-line comments must not begin with /** (double asterisk) as the parser might mistake it for a DocBlock. Use /* (single asterisk) instead.
>
> [Source](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments)

```suggestion
/* 
 * These constants were created to simplify the process when this page gets
 * moved to a React-based approach. It was a choice to not refactor any
 * further than this.
 */
```

Reply:
Addressed in dc420c8.